C/services/common/include/notification_manager.h

```diff
@@ -209,6 +209,7 @@ class NotificationInstance
 		time_t			m_lastSent;
 		NotificationState	m_state;
 		bool			m_zombie;
+		bool			m_doSend;
```

Comment:
If we don't use the repeatFrequency in Toggled that's not needed at all.

If we do, we need to check wether a "triggered" notifications was sent before sending the "cleared" notification.

After sending the first ""cleared" notification the state can change to "Triggered" but we don0't send the notification because of repeatFrequency.

If state changes to "Cleared" we need to know wether to send the notification or not.